### PR TITLE
Improved code of order page by replacing list header with card header to display heading of cards(#2893v1u)

### LINKED ISSUE #144: Replace header of all cards from list header to card header on orders page
## Current behavior
Currently, we have used a list header for displaying the header of the card but it is not necessary every time card contains a list.

## Expected behavior
The card header component should be used as header of the cards.

## Steps to reproduce the issue
Replace list header with card header component.

## Can you handle fixing this bug by yourself?

- [X] YES
- [ ] NO

## Environment details

- Browser: 
- OS: 
- Code Version:

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#144 

Closes #144

### Short Description and Why It's Useful
 We previously used a list header for displaying the header of the card but it is not necessary every time card contains a list. So in this PR list header is replaced with card-header

### Screenshots of Visual Changes before/after (If There Are Any)
localhost_8100_order_NN13435

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/hotwax/commerce-hub#contribution-guideline)

### CODE REVIEW

File: src/views/Order.vue [hunk 1]
```diff
@@ -350,6 +360,12 @@ ion-select {
   margin-inline-start: 0;
 }
 
+ion-card-header {
```
Comment (dt2patel) [inline src/views/Order.vue]: this is a global change. Need to update all pages and make rule global.

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/Order.vue [hunk 2]
```diff
@@ -48,15 +48,15 @@
 
           <div class="info">
             <ion-card>
+              <ion-card-header>
+                <ion-card-title>{{ order.customer?.name }}</ion-card-title>
+                <!-- TODO: handle this property to display loyalty options -->
+                <ion-chip v-if="order.customer?.loyaltyOptions">
```
Comment (dt2patel) [inline src/views/Order.vue]: Please check figma and match chip styling

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/Order.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Order.vue]: Ok Sir, I have made this rule global.

File: src/views/Order.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Order.vue]: I have applied outline to this chip according to the figma design.

File: src/views/Order.vue [hunk 3]
```diff
@@ -240,7 +251,29 @@
                       </ion-buttons> -->
                     </ion-list>
                   </ion-card>
-                  
+                  <ion-card>
+                    <ion-card-header>
+                      <ion-card-title>{{ $t("Fulfillment") }}</ion-card-title>
```
Comment (dt2patel) [inline src/views/Order.vue]: why is the entire card being added here? The expected change should just be the addition of the card header right?

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/Order.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/Order.vue]: I think it was pushed by mistake during resolving a merge conflict. I have removed the extra code and updated the PR.

**Review by dt2patel — APPROVED** [review]

**Pull request merged**